src/helpers/LinkButtonComponentType.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * The proptypes for {@link LinkButtonComponentType}.
+ *
+ * There is an additional required prop missing from this interface,
+ * `onKeyDown`, which follows the usual React interface
+ * `React.KeyboardEventHandler`.
+ */
```

Comment:
We should also document that it needs to execute `onClick` or handle `onKeyDown` to execute on pressing space.